Sofa/Component/SolidMechanics/FEM/Elastic/src/sofa/component/solidmechanics/fem/elastic/TriangleFEMForceField.h

```diff
@@ -137,15 +136,9 @@ class TriangleFEMForceField : public core::behavior::ForceField<DataTypes>
     Data<bool> f_planeStrain;
 
     Data<std::string> d_method; ///< large: large displacements, small: small displacements
-    Data<Real> d_poisson; ///< Poisson ratio in Hooke's law
```

Comment:
Lifecycle for the old data (f_poisson and f_young) should be modified to removed data.

Reply:
Indeed...

Reply:
ah no indeed actually. The rename is very recent and does not overlap a release. The old Data (`f_poisson`) takes care of the deprecation. About the Young's modulus, it's another type (from Real to VecReal), so we cannot set a deprecation (that is why it is breaking).

Reply:
Ah yes you are right, I am sorry. In this case there is nothing particular we can do I guess...